src/RulesMD/RulesMD.csproj

```diff
@@ -50,6 +50,16 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="CLIOptions.cs" />
+    <Compile Include="obj\Debug\\MarkdownCreator.cs">
```

Comment:
Lines 53-62 are why the release build is failing. They're hardcoded to debug--and they seem to be redundant with lines 65-69. I think you want to remove lines 53-62 and possibly change line 67 to set DesignTime to True